code/CLI/CLIEditor.py

```diff
@@ -239,11 +587,157 @@ def REPL():
+        response = command.execute()
+        # ensure there was a response
+        if response:
+            status, msg = response
+            # ensure command was successful
+            if status:
+                print(f"{SUCCESS_COLOR}SUCCESS:{NORMAL_COLOR} {msg}")
+            else:
+                print(f"{ERROR_COLOR}ERROR:{NORMAL_COLOR} {msg}")
+
+    def do_load_model(self, args):
+        words = args.split()
+        # grab the command
+        # this handles the case where there are no arguments
+        command = getCommand(model, "load_model", words[0:])
+        # execute the command
+        response = command.execute()
+        # ensure there was a response
+        if response:
+            status, msg = response
+            # ensure command was successful
+            if status:
+                print(f"{SUCCESS_COLOR}SUCCESS:{NORMAL_COLOR} {msg}")
+            else:
+                print(f"{ERROR_COLOR}ERROR:{NORMAL_COLOR} {msg}")
+
+    def do_undo(self, args):
+        words = args.split()
+        # grab the command
+        # this handles the case where there are no arguments
+        command = getCommand(model, "undo", words[0:])
+        # execute the command
+        response = command.execute()
+        # ensure there was a response
+        if response:
+            status, msg = response
+            # ensure command was successful
+            if status:
+                print(f"{SUCCESS_COLOR}SUCCESS:{NORMAL_COLOR} {msg}")
+            else:
+                print(f"{ERROR_COLOR}ERROR:{NORMAL_COLOR} {msg}")
+
+    def do_redo(self, args):
+        words = args.split()
+        # grab the command
+        # this handles the case where there are no arguments
+        command = getCommand(model, "redo", words[0:])
+        # execute the command
+        response = command.execute()
+        # ensure there was a response
+        if response:
+            status, msg = response
+            # ensure command was successful
+            if status:
+                print(f"{SUCCESS_COLOR}SUCCESS:{NORMAL_COLOR} {msg}")
+            else:
+                print(f"{ERROR_COLOR}ERROR:{NORMAL_COLOR} {msg}")
+
+    def do_help(self, args):
+        if args:
+            print_help_message(model, args)
+        else:
+            print_help_message(model, "")
+
+    def do_exit(self, args):
+        prompt_exit(model)
+
+    def complete_help(self, text, line, begidx, endidx):
+        commands = list(CommandData.COMMANDS.keys())
+        if text:
+            return [
+                command for command in commands
+                if command.startswith(text)
+            ]
+        else:
+            return commands
+    
+##########################################################################
+
+def runCMD():
+    REPL().cmdloop()
 
 ##########################################################################
 
 # Program runs the REPL by default 
 if __name__ == "__main__":
-    REPL()
+    REPL().cmdloop()
```

Comment:
Maybe this should call runCMD() ? So that if we change runCMD() then we wouldn't have to change this line either